apps/backend/package.json

```diff
@@ -20,6 +20,7 @@
     "start": "node dist/src/main",
     "start:debug": "nest start --debug --watch",
     "start:dev": "nest start --watch",
+    "start:fips": "node --force-fips dist/src/main",
```

Comment:
Forcing node to use fips compliant algorithms will cause nodejs to use the system provided cryptography module, in this case openssl because that is the fips approved module on the image used to build Heimdall, in this case ubi8. This requires the use of native api